config/resgest_ci.json5

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Do we need this file? and if so, I suppose it should be named `regtest` 😅

Reply:
no idea where this came from. Deleted this one.

Reply:
it looks like it wasn't necessary, so I removed it.